error-prone-contrib/src/test/java/tech/picnic/errorprone/refasterrules/StringRulesRecipesTest.java

```diff
@@ -0,0 +1,50 @@
+package tech.picnic.errorprone.refasterrules;
+
+import static org.openrewrite.java.Assertions.java;
+
+import com.google.errorprone.FileObjects;
+import java.io.IOException;
+import javax.tools.JavaFileObject;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+final class StringRulesRecipesTest implements RewriteTest {
+  @Override
+  public void defaults(RecipeSpec spec) {
+    spec.recipe(new StringRulesRecipes());
+  }
+
+  @Test
+  void stringValueOf() {
+    // XXX: Use text blocks once supported.
+    rewriteRun(
+        // language=java
```

Comment:
Ah yes I've looked at this in the past; we already have [`@Language("java")` on those `java(...)` methods](https://github.com/openrewrite/rewrite/blob/6f64c1166ec8fea0468a577e62fc6a2543efb8a9/rewrite-java/src/main/java/org/openrewrite/java/Assertions.java#L92-L95), but IntelliJ does not pick those up when using `rewrite-java` as a library. I'd wondered if their [CLASS retention policy](https://docs.oracle.com/javase/8/docs/api/java/lang/annotation/RetentionPolicy.html#CLASS) on the[`@Language` annotation](https://github.com/JetBrains/java-annotations/blob/b41de54f50e463211c71435c0fd86edeb9e3a152/common/src/main/java/org/intellij/lang/annotations/Language.java#L45-L48) might factor in. Suggestions welcome!

Reply:
Thanks for the hint! Looks configurable indeed, although it's a shame to have to replicate what's also already configured in the methods parameters themselves.. 🤔 
![image](https://github.com/PicnicSupermarket/error-prone-support/assets/1027334/a681a578-7680-4569-bd24-731050330af5)

Reply:
I didn't know about `@Language`, but one would expect that to do the right thing indeed :thinking:. And since IDEA has access to the bytecode, `CLASS` retention should suffice :thinking:

Reply:
Logged an issue here: https://youtrack.jetbrains.com/issue/IDEA-341219/Library-method-arguments-annotated-with-Language-not-highlighted

Reply:
Nice; gave it a thumbs-up ;)